profiles/base/kustomization.yaml

```diff
@@ -30,6 +31,13 @@ vars:
     apiVersion: v1
   fieldref:
     fieldpath: data.admin
+- name: gcp-sa
```

Comment:
I think this will cause an error if gcp-sa isn't explicitly defined in the parameters.
Forcing every KFDef to explicitly set GCP specific parameters to the empty string doesn't seem scalable.

Reply:
The default value is empty string, if param not set, the plugin will be ignored.